frontend/appflowy_flutter/lib/workspace/presentation/home/menu/view/draggable_view_item.dart

```diff
@@ -65,6 +65,11 @@ class _DraggableViewItemState extends State<DraggableViewItem> {
       onMove: (data) {
         final renderBox = context.findRenderObject() as RenderBox;
         final offset = renderBox.globalToLocal(data.offset);
+
+        if (offset.dx > renderBox.size.width) {
```

Comment:
Is it to cancel the highlight status of the view item?

Reply:
It's because it will try to compute the drop of the view item in the sidebar, despite it not being relevant (outside of the boundary).

Without this part, when you drag and drop a view item, it can disappear from sidebar or be moved somewhere unexpected.